src/api/__tests__/Feed.test.js

```diff
@@ -98,6 +99,11 @@ const versionsWithCurrent = {
     entries: [mockCurrentVersion, mockFirstVersion, deleted_version],
 };
 
+const annotations = {
+    total_count: 1,
```

Comment:
Does this get returned by the API or is it added client-side? It's a marker-based API, so it doesn't usually know the total item count.